src/pages/_includes/graphql/catalog-service/categories.md

```diff
@@ -0,0 +1,33 @@
+The `categories` return object is an interface that can contain the following fields. It is implemented by the [`CategoryView`](#categoryview-type) and [`CategoryViewInterface`](#categoryviewinterface-attributes) types.
```

Comment:
There's no return objected named `categories`. You can say the `categories` query returns the `CategoryView` object, which implements `CategoryViewInterface`. I think I would mention that `children` is not defined in the interface.